modules/ROOT/pages/about-my-applications.adoc

```diff
@@ -9,17 +9,37 @@ applications that hold a contract between an API and a user.
 
 image::ex2-my-applications.png[]
 
-A contract holds information about the contract with a client ID and client secret for the API.
-Each application lists a description, its URL, redirect URIs, grant types, and a usage dashboard.
+A contract holds information including a client ID and optional client secret
+for the API. Each application lists a description, its URL, redirect URIs,
+grant types, and a usage dashboard.
 
-The usage information lets you set the verification duration. If not specified, the range is the last 8 days. The usage lists the total number of requests to the API, average latency, the error rate, and graphs for total requests, average latency, requests by HTTP status code, and the error percentage.
-
-The applications result when users click Request Access to establish
-a link between an application and an API. Either the application already exists,
-or the user creates the application from the Request Access menu.
+The usage information enables you to set the verification duration. If not specified,
+the range is the previous eight days. The usage lists the total number of requests to
+the API, average latency, the error rate, and graphs for total requests,
+requests by HTTP status code, and the error percentage.
```

Comment:
the range is the previous eight days. The usage lists total number of requests to
the API, average latency, error rate, and graphs for total requests,
requests by HTTP status code, and error percentage.

Reply:
Why is each line its own paragraph (+)? 
I see this method, and then I also see a method in which the + appears only at the beginning of a new paragraph.
Would you mind sending me a separate Slack message or email to explain?

Reply:
Please see: https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#breaks